Src/Couchbase.Linq/QueryGeneration/N1QLExpressionTreeVisitor.cs

```diff
@@ -684,5 +701,31 @@ protected override Expression VisitSubQueryExpression(SubQueryExpression express
 
             return expression;
         }
+
+        //public Expression VisitAndEnsureDateTimeFormat(Expression expression, N1QlDateTimeFormat expectedDateTimeFormat)
+        //{
+        //    if ((expression.Type != typeof (DateTime)) && (expression.Type != typeof (DateTime?)))
+        //    {
+        //        return VisitExpression(expression);
+        //    }
+
+        //    var result = expression;
+
+        //    if ((expectedDateTimeFormat == N1QlDateTimeFormat.IsoString) &&
+        //        (DateTimeFormat == N1QlDateTimeFormat.UnixMilliseconds))
+        //    {
+        //        _expression.Append("STR_TO_MILLIS(");
+        //        result = VisitExpression(expression);
+        //        _expression.Append(')');
+
+        //        DateTimeFormat = N1QlDateTimeFormat.IsoString;
+        //    }
+        //    else if ((expectedDateTimeFormat == N1QlDateTimeFormat.UnixMilliseconds))
+        //    {
+                
+        //    }
+
+        //    return result;
+        //}
     }
```

Comment:
Oops, you're right.  Old experiment I forgot to clean up.  I should be able
to get it to you this afternoon.  That's what I get for working too late at
night.  :)

Brant

On Fri, Oct 2, 2015 at 2:15 PM, Jeffry Morris <notifications@github.com>
wrote:

> In Src/Couchbase.Linq/QueryGeneration/N1QLExpressionTreeVisitor.cs
> <https://github.com/couchbaselabs/Linq2Couchbase/pull/42#discussion_r41050708>
> :
>
> > +        //    if ((expectedDateTimeFormat == N1QlDateTimeFormat.IsoString) &&
> > +        //        (DateTimeFormat == N1QlDateTimeFormat.UnixMilliseconds))
> > +        //    {
> > +        //        _expression.Append("STR_TO_MILLIS(");
> > +        //        result = VisitExpression(expression);
> > +        //        _expression.Append(')');
> > +
> > +        //        DateTimeFormat = N1QlDateTimeFormat.IsoString;
> > +        //    }
> > +        //    else if ((expectedDateTimeFormat == N1QlDateTimeFormat.UnixMilliseconds))
> > +        //    {
> > +
> > +        //    }
> > +
> > +        //    return result;
> > +        //}
> >      }
>
> This should be removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/couchbaselabs/Linq2Couchbase/pull/42/files#r41050708>.
>



-- 
Thanks,
Brant Burnett
CenterEdge Software
(336) 598-5934
http://centeredgesoftware.com